eessi_bot_event_handler.py

```diff
@@ -35,6 +36,9 @@ def handle_issue_comment_event(self, event_info, log_file=None):
         comment_author = request_body['comment']['user']['login']
         comment_txt = request_body['comment']['body']
         log("Comment posted in %s by @%s: %s" % (issue_url, comment_author, comment_txt))
+        # if bot instance responsible for control center, process event
+        if False:
```

Comment:
I would do the `if` in the function itself, where we can also log a message that we got an event, but that we're not doing anything with it because this bot instance is not configured to do so?